website/docs/r/pi_network_address_group.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_pi_network_address_group"
+description: |-
+  Manages network address group.
+subcategory: "Power Systems"
+---
+
+# ibm_pi_network_address_group
+
+Create, update, and delete a network address group.
+
+## Example Usage
+
+The following example creates a network address group.
+
+```terraform
+    resource "ibm_pi_network_address_group" "network_address_group_instance" {
+        pi_cloud_instance_id = "<value of the cloud_instance_id>"
+        pi_name = "name"
+    }
+```
+
+### Notes
+
+- Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+- If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  - `region` - `lon`
+  - `zone` - `lon04`
+  
+Example usage:
+
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+## Timeouts
+
+ibm_pi_network_address_group provides the following [timeouts](https://www.terraform.io/docs/language/resources/syntax.html) configuration options:
+
+- **delete** - (Default 5 minutes) Used for deleting network address group.
```

Comment:
Add update and create timeout when added to resource.

Reply:
Not needed here